Tests/Common/Util/StringAnyEncodable.swift

```diff
@@ -9,13 +9,15 @@ struct DummyData: Codable, Equatable {
     let testValue: String
     let dict: [String: String]
     let array: [Int]
+    let hashables: [String: [String]]
```

Comment:
Could you explain the name `hashables` and why this data type is important to test? Are you trying to test nested lists? If so, maybe use the name `hashableNestedList`? 

I bring this up because from understanding the bug we are trying to fix, I was expecting to see us test nesting but the test function doesn't include "nest" in the naming?

Reply:
I [responded above](https://github.com/customerio/customerio-ios/pull/358#discussion_r1273337734) to another comment which might help, let me know if you have more questions.